src/client/app/shared/actions/action-creators/game.action-creator.ts

```diff
@@ -5,7 +5,7 @@ import {
 } from '../game.actions';
 
 export const GameActions = {
-  completeGame(time: number, text: string): Action {
+  completeGame(time: any, text: string): Action {
```

Comment:
Would you switch it back to `number` and pass a timestamp with `Date.now()`, in order to keep it statically typed.